pkg/kv/kvpb/errors.go

```diff
@@ -305,6 +306,58 @@ const (
 	NumErrors int = 45
 )
 
+// TODO: figure out whether we need this type migration, and if we do, how to
+// handle the CI failures.
```

Comment:
The migrations here look correct. What are the CI failures?

Reply:
There were a handful https://teamcity.cockroachdb.com/viewLog.html?buildId=8747339&buildTypeId=Cockroach_BazelEssentialCi. But I think it was only a couple of errors for which we defined custom wrapper encoders / decoders, and there was an incorrect order of initialization (i.e. the type migration previously would apply after the registration of encoders). I just pushed an update that should fix those errors.

Reply:
Indeed, that was the issue, so I updated the commit. RFAL.